pkg/internal/resources/catapult/catapult.go

```diff
@@ -294,9 +299,6 @@ func Manifests(c Config) ([]unstructured.Unstructured, error) {
 		if labels == nil {
 			labels = map[string]string{}
 		}
-		labels[constants.NameLabel] = "catapult"
-		labels[constants.InstanceLabel] = c.Name
-		labels[constants.ManagedbyLabel] = constants.ManagedbyKubeCarrierOperator
```

Comment:
These lines cannot be removed, otherwise, some objects will not have common labels, because not all objects are created from `MkLayer`
You can check the changes of `catapult.golden.yaml`, and the common labels are removed from a `ClusterRole` which we created after the `MkLayer`

Reply:
Ok re-added that code